regen-network/regen-web
# fix: Do not display project forms if trying to access them for a project that does not exist or not admin of

## Description

Closes: regen-network/regen-registry#1714



---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...


### How to test

Go to https://deploy-preview-1984--regen-registry.netlify.app and login.
Go to `https://deploy-preview-1984--regen-registry.netlify.app/project-pages/${offChainProjectId}/${step}` and `https://deploy-preview-1984--regen-registry.netlify.app/project-pages/${onChainProjectId}/edit/${step}` where `step` in `choose-credit-class` (only available for the "Create" path, not the "Edit" path), `basic-info`, `location`, `roles`, `description`, `media` or `metadata`
You should test that for all cases:
- if project with `offChainProjectId` or `onChainProjectId` doesn't exist, it displays 404 page
- if you're not an admin of the project with `offChainProjectId` or `onChainProjectId`, it displays access denied

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...
